### PR TITLE
Fix routes

### PR DESCRIPTION
**Pull request by Ostend**
-React app typo in routes fixed
-Disconnected the broken mongodb app
-Express and React can communicate with these changes
-added readme template
-moved meeting notes to blog.md
-add more parameters to the .gitignore

To do next: 
-change structure of express app
-recreate DB

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,150 @@
...
+<!-- PROJECT SHIELDS -->
+<!--
+*** I'm using markdown "reference style" links for readability.
+*** Reference links are enclosed in brackets [ ] instead of parentheses ( ).
+*** See the bottom of this document for the declaration of the reference variables
+*** for contributors-url, forks-url, etc. This is an optional, concise syntax you may use.
+*** https://www.markdownguide.org/basic-syntax/#reference-style-links
+-->
+
+
```
Comment (rodgobbi) [inline README.md:25]: The README was made with a template, wasn't it?
We can remove the extra text that we are not gonna use.

**Review by rodgobbi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Ostend) [reply README.md:25]: hello, yes it was a template- I just copy pasted and left the text so everyone can see where I got it from, and also show the options of what can be added in the readme.. Im still getting to know the project, so did not want to be the only one changing and making decision about the readme.

Comment (Moomia) [reply README.md:25]: We could copy paste the text to our Notion into a subpage and keep it there for reference, hm?

Comment (Ostend) [reply README.md:25]: that sounds good!

**Pull request merged**